### DISCUSSION TITLE
Problema con il registro

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#590] [Richieste di aiuto]
Buongiorno io sono passato al secondo quadrimestre ma appena ho inserito la data di oggi, ogi non posso ne girmare né mettere voti

**Comment by trinko** [comment]
Cosa intendi per "sono passato al secnodo quadrimestre"? Che modifiche hai fatto?

**Comment by Davide031** [comment]
Ho impostato nei parametri fine primo periodo e ho messo oggi

**Comment by trinko** [comment]
Il giorno finale è l'ultimo del periodo: il secondo quadrimestre inizia quindi dal giorno successivo.
Comunque il fatto che non possa firmare le lezioni è probabilmente dovuto al fatto che hai svolto lo scrutinio dello classe (o almeno aperto lo scrutinio): questo blocca ogni modifica sul registro per il periodo dello scrutinio.
